### DISCUSSION TITLE
[A-L] (2023/24) Foglio 8 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#155] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-17 184712](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/6269a0a7-472f-495f-a4e3-742660a7ad19)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/261ad262-a7e7-4d49-a628-1b93b785bf11)

**Reply by CiottoloMaggico** [reply]
Credo che questa dimostrazione non sia conclusiva, il T.Lagrange ci dice che se esiste un sottogruppo H del gruppo G allora la cardinalità di H divide G tuttavia potrebbero non esistere sottogruppi di cardinalità H anche se H divide G.
 Considera il gruppo alterno A_4, anche se il suo ordine è 12  non esistono sottogruppi di ordine 6 nonostante 6/12

**Comment by cornflexxx** [comment]
Per il Teorema di Cauchy sappiamo che esistono sottogruppi di $S_{30}$ di ordine primo , in particolare  chiameremo $G,H$, rispettivamente $o(G)=11$ e $o(H)=19$. Dato che $o(G),o(H)|o(S_{30})$ (come visto sopra), otteniamo due sgr. di $S_{30}$  di ordine primo , quindi ciclici.
Essendo $209=19\cdot 11$ possiamo provare  a moltiplicare i due sottogruppi, e vedere se il loro prodotto è effettivamente un sottogruppo. 

Definiamo $GH=$ { $gh  | g\in G,h\in H$ } , per prima cosa dobbiamo dimostrare la chiusura rispetto al 'prodotto' $gh(g'h')\in GH$, dove $g,g'\in G $ e $h,h' \in H$, sappiamo però che i due gruppi sono ciclici e quindi abeliani allora: $gh(g'h') =(gg')(hh')$ , quindi $(gg')=g''\in G$ e $(hh')=h'' \in H$ $\longrightarrow g''h''\in GH$. Inoltre  abbiamo scoperto (visto che si tratta di prodotto di gruppi abeliani) che $GH=HG$. 
Verifichiamo che l'inversa appartiene  ancora a $GH$, quindi $(gh)^-1 \in GH$: $(gh)^-1=h^{-1}  g^{-1}\in HG=GH$

Allora $GH$ è un sottogruppo di $S_{30}$  e ha ordine pari al numero di elementi in $G$  per il numero di elementi in $H$(tutte le combinazioni tra gli elementi), a meno di duplicati , infatti definendo $GH$ attingiamo da due sottogruppi diversi ,ma comunque sottoinsiemi di una stessa collezione. In questo caso siamo fortunati e possiamo ignorare il problema perché $o(G)+o(H)=11+19=30$ , allora possiamo possiamo scegliere due sottoinsiemi disgiunti di elementi. Quindi $o(GH)=o(G)\cdot o(H)=11\cdot 19=209$.

**Reply by CiottoloMaggico** [reply]
Forse mi sfugge qualcosa, però un gruppo simmetrico di ordine >= 3 ha il centro uguale al solo elemento neutro. Non capisco come G e H possano essere sottogruppi di S_30 abeliani

**Reply by cornflexxx** [reply]
Un sottogruppo di ordine primo è sempre ciclico e ogni gruppo ciclico è abeliano
